app/code/RefreshMarkdownTask.php

```diff
@@ -30,13 +30,12 @@ class RefreshMarkdownTask extends BuildTask
      */
     public function run($request)
     {
```

Comment:
Add a class scope variable $this->request = $request; to capture the passed in request and use it later on in the class.